packages/telemetry/package.json

```diff
@@ -107,7 +107,7 @@
     "@angular/core": "19.2.15",
     "@angular/platform-browser": "19.2.15",
     "@angular/router": "19.2.15",
-    "@firebase/app": "0.14.4",
+    "@firebase/app": "0.14.6",
```

Comment:
Rebase with telemetry, made this change upstream since I ran into it as well in my PR.